src/util/cached-prop-defs.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import assign from './assign';
+import dashCase from './dash-case';
+import getAllKeys from './get-all-keys';
+
+const CACHED_PROP_DEFS:string = '____skate_propDefs';
```

Comment:
I'm sorry, this comment will be maybe off topic, but I have to react on this one :D

> For skatejs right now Flow is a better choice.

Who is judge and jury of that? you @andreawyss ? :)

> I used TypeScript for 2 years but now for functional programming I prefer Flow.

Why a sudden change of heart? Typescript is super powerful for functional stuff, maybe you didn't understood something ? ( TS is not strictly OOP at all )

> No extra compiler needed, just annotate your JS or ES6 code. It is closer to the platform.

No Extra compiler needed? Typescript is both compiler and type checker, so you don't need babel anymore. 
What is closer to the platform? With Flow you are not writing valid javascript ( by .js extension ), Flow is not even written with the platform language :)

Also Typescript: 
- runs flawlessly in hybrid mode ( js, ts files + no babel needed )
- has very strong and powerful type inference
- mature and large ecosystem and works with all IDE's and Editors
- it gives library consumers extra benefit by providing up to date typings for free ( no more manualy crating `.d.ts` files

Don't get me wrong, I say better any type checker than none, but probably the SkateJs team is in charge to properly consider what's the best for them.